pkg/cri/config/config.go

```diff
@@ -210,7 +210,7 @@ type Registry struct {
 	Mirrors map[string]Mirror `toml:"mirrors" json:"mirrors"`
 	// Configs are configs for each registry.
 	// The key is the domain name or IP of the registry.
-	// This option will be fully deprecated for ConfigPath in the future.
+	// DEPRECATED: Use ConfigPath instead.
```

Comment:
(This has been already deprecated since v1.5)
https://github.com/containerd/containerd/blob/1f236dc57aff44eafd95b3def26683a235b97241/releases/v1.5.0.toml#L133